labelbox/schema/task.py

```diff
@@ -102,6 +101,19 @@ def result(self) -> List[Dict[str, Any]]:
                 'global_key': data_row.get('globalKey'),
             } for data_row in result['createdDataRows']]
 
+    @property
+    def failed_data_rows(self) -> List[Dict[str, Any]]:
```

Comment:
Glad we're introducing this partial failure concept!
One concern I have is, how will the customer know whether to call `failed_data_rows`. They wouldn't know that some data rows failed to be imported unless they explicitly called `Task.failed_data_rows`.
I think we should either:
1. Include failed data rows as part of `Task.result()`
2. Or include a warning message when they call `Task.result()`, if the result contains `errors`
(cc @olegtrygub )

Reply:
Also another question--if all data rows failed to be imported, will the Task status be "FAILED"?

Reply:
Yeah I originally had wanted to include in `Task.result()` but realized if I did I would be breaking the interface of that property since it's just an array of data rows today. Ideally I'd return this in an object on the result that would look something like this.
```json
{
  "createdDataRows": [],
  "failedDataRows": []
}
```

Reply:
@kkim-labelbox in response to your second question, currently the API does not mark that scenario as "FAILED" but I'll make a change to introduce that.

Reply:
We'll only show the task as failed if all of the data rows have failed to be created, otherwise it'll be partial and the task will be marked as completed. In terms of showing the warning or the failed rows as part of the result, is it okay to introduce a breaking change here to the interface of the currently returned result?

Reply:
Ah yeah if the `Task.result()` is currently only returning the list of data rows, I don't see a good way to include failed data rows as part of it, without making a breaking change. 

At the same time, I feel that adding a new property may be confusing to the users. They may ask why there's `Task.errors` but also `Task.failed_data_rows`. They'd have to understand the nuances between a failed Task (all data rows failed) versus a completed Task with a partial failure. My proposal is to modify the `Task.errors` property to return the failed data rows when `status == "COMPLETE"` but result['error'] exists, and add an additional condition to check for `status == "COMPLETE" and result['errors'] is not None` to throw the existing warning message - https://github.com/Labelbox/labelbox-python/blob/20ee3a36149ede07ca0c24d23d6751ff6953a0fe/labelbox/schema/task.py#L68

What do you guys think?

Reply:
I like that proposal, thanks @kkim-labelbox !!